WordPress/Classes/ViewRelated/Me/App Settings/AppSettingsViewController.swift

```diff
@@ -257,6 +259,14 @@ class AppSettingsViewController: UITableViewController {
         }
     }
 
+    func pushAppColorList() -> ImmuTableAction {
+        return { [weak self] row in
+            let vc = UIHostingController(rootView: AppColorListView())
```

Comment:
This is more of a nitpick and not a huge deal in this context since I know what this is but we try to avoid abbreviations when naming something. We have a Swift style guide [here](https://github.com/wordpress-mobile/swift-style-guide) (although, it looks like it hasn't been updated in quite a while 😅) however it doesn't mention this explicitly. It can be found in one of the [links](https://www.swift.org/documentation/api-design-guidelines/#use-terminology-well) from that document.

Reply:
Oh, I was thinking that `vc` is widely used abbreviation + also saw it in some other places in the codebase so I assumed it's fine here, but no worries, I'll just rename this one in the next commit.